test_api.py

```diff
@@ -0,0 +1,72 @@
+import os
+import unittest
+from app import app
+
+class ApiEndPointsTest(unittest.TestCase):
+
+	def test_base_route(self):
+		self.test_app = app.test_client()
+		response = self.test_app.get('/')
+		self.assertEquals(response.status_code,200)
+
+	def test_addToMenu(self):
```

Comment:
Python doesn't use camelcase for naming of methods and functions.
Only for classes.
Revert accordingly